liblz4_vendor/cmake/Modules/Findlz4.cmake

```diff
@@ -4,6 +4,13 @@ if(NOT lz4_ROOT_DIR AND DEFINED ENV{lz4_ROOT_DIR})
       "lz4 base directory location (optional, used for nonstandard installation paths)")
 endif()
 
+# When using lz4 from conda on Windows, and cmake before 3.29.0,
+# we need to add the library prefix "lib" so that the library can
+# be successfully found.
+if(MSVC AND CMAKE_VERSION VERSION_LESS "3.29.0")
+  set(CMAKE_FIND_LIBRARY_PREFIXES "" "lib")
+endif()
```

Comment:
I'm guessing this works fine on Windows without Conda?

Reply:
> I'm guessing this works fine on Windows without Conda?

Not exactly.  With our current setup, we don't actually install liblz4 anywhere, and we just always build it.  But since we are trying to be more like a Linux distribution with Conda, we prefer to use the system version there.  So this would have always failed with liblz4 (at least, prior to CMake 3.29.0), we just never noticed before.